checks/containers/container_engine/xccl_tests.py

```diff
@@ -77,7 +77,9 @@ def set_perf(self):
 @rfm.simple_test
 class NCCLTestsCE(XCCLTestBase):
     valid_systems = ['+ce +nvgpu']
-    image_tag = parameter(['cuda12.8.1'])
+
+    # FIXME update the image tag once the port issue is fixed
```

Comment:
do you mean https://github.com/eth-cscs/cscs-reframe-tests/pull/343 ?
do we still need this change ?